project.ptx

```diff
@@ -1,106 +1,48 @@
-      <format>pdf</format>
-      <source>source/main.ptx</source>
-      <publication>publication/publication.ptx</publication>
-      <output-dir>output/pdf</output-dir>
-      <xsl>xsl/acs-latex.xsl</xsl>
-    </target>
-    <target name="solutions" pdf-method="xelatex">
-      <format>pdf</format>
-      <source>source/acs-solution-manual.ptx</source>
-      <publication>publication/publication-solutions.ptx</publication>
-      <output-dir>output/solutions-pdf</output-dir>
-      <xsl>xsl/acs-solution-manual.xsl</xsl>      
-    </target>
-    <target name="solutions-latex">
-      <format>latex</format>
-      <source>source/acs-solution-manual.ptx</source>
-      <publication>publication/publication-solutions.ptx</publication>
-      <output-dir>output/solutions-latex</output-dir>
-      <xsl>xsl/acs-solution-manual.xsl</xsl>
-    </target>
-    <target name="workbook14" pdf-method="xelatex">
-      <format>pdf</format>
-      <source>source/acs-activity-workbook-14.xml</source>
-      <publication>publication/publication-workbook.ptx</publication>
-      <output-dir>output/workbook-pdf</output-dir>
-      <xsl>xsl/acs-activity-workbook.xsl</xsl>      
-    </target>
-    <target name="workbook14-latex">
-      <format>latex</format>
-      <source>source/acs-activity-workbook-14.xml</source>
-      <publication>publication/publication-workbook-14.ptx</publication>
-      <output-dir>output/workbook-latex</output-dir>
-      <xsl>xsl/acs-activity-workbook.xsl</xsl>
-    </target>
-    <target name="workbook58" pdf-method="xelatex">
-      <format>pdf</format>
-      <source>source/acs-activity-workbook-58.xml</source>
-      <publication>publication/publication-workbook-58.ptx</publication>
-      <output-dir>output/workbook-pdf</output-dir>
-      <xsl>xsl/acs-activity-workbook.xsl</xsl>      
-    </target>
-    <target name="workbook58-latex">
-      <format>latex</format>
-      <source>source/acs-activity-workbook-58.xml</source>
-      <publication>publication/publication-workbook-58.ptx</publication>
-      <output-dir>output/workbook-latex</output-dir>
-      <xsl>xsl/acs-activity-workbook.xsl</xsl>
-    </target>
-
-    <target name="subset">
-      <format>html</format>
-      <source>source/main.ptx</source>
-      <publication>publication/publication.ptx</publication>
-      <output-dir>output/subset</output-dir>
-      <stringparam key="debug.skip-knowls" value="yes"/>
-      <!-- edit this to change the section/chapter/etc. to include
-           in your subset build. Use the xml:id, not the file name! -->
-      <xmlid-root>C-8</xmlid-root>
-      <xsl>xsl/acs-html.xsl</xsl>
-    </target>
+    <target name="html" format="html" xsl="acs-html.xsl" />
+    <target name="latex" format="latex" xsl="acs-html.xsl" />
+    <target name="pdf" format="pdf" xsl="acs-html.xsl" />
+    <target 
+      name="html-hosted"
+      format="html"
+      xsl="acs-html.xsl"
+      output-dir="/home/activec2/public_html/single" />
```

Comment:
This configuration is probably only relevant to a single machine on earth. There's a better way to modify the `output-dir` ad hoc in that instance, by writing a build script in Python rather than checking in a machine-specific configuration into your repo:

```python
t = Project.parse().get_target(name="html")
t.deploy_dir = "/home/activec2/public_html/single"
t.build()
```

Reply:
This is what we're using to build what is posted on activecalculus.org. We don't have a lot of control over what happens on our hosting server, so I would prefer not to have to have some sort of custom build script and want this build target left in place.

Reply:
I think it should be quite simple to make this change to your deployment workflow, but it's also not really that critical an issue. I will instead move it to the bottom of the target list and leave a comment so it won't distract a potential contributor.